docs/doc-guideline.md

```diff
@@ -0,0 +1,52 @@
+# DOC GUIDELINE
+
+## How to write an article for the website
+
+1. All the problems written should necessarily include 3 sections
+   1. Explanation of the article
+   2. Sample input
+   3. Sample output
+
+2. All problems should be supplemented by examples of code. 
+
+3. Every problem should have two or more examples with at least one of them well-explained.
+
+4. Time complexity and space complexity should be included, if applicable.
+
+5. Formulae or any important point should be highlighted.
+
+## GUIDELINES FOR THE CODE
+The guidelines for the code:
+   1.  The width of text in each line should not exceed 60 characters
+
+   2.  All documentations should be written using multi-line comments 
+
+   3. Comments should be written above the code and not beside it.
+
+   4. Function and variable names should follow camelcasing. Example: uglyNumbers()
+
+   5. Indentation should be done using 4 spaces.
+
+   6. In case of the code being written in multiple languages, the output of all the codes should be the same
+
+   7. There should not be any spaces for function call or function declaration
+
+   8. Avoid the use of typedef
+
+   9. Avoid the use of static and global variables
+
+   10. When using cout, use a space between cout and "<<".
+
+   11. There should be space after comma in parameter passing and declaration list
+
+   12. In the beginning of each code, use multiline comments to explain the purpose of the program. Make the points bulleted and short.
+
+# Reviewing Process
+
+1. Submit the article for review after writing it.
+
+2. The team will review your article and give you feedback for the article
+
+3. Work upon the feedback and improve your article. Submit it again for further review.
+
+4. If all the required improvements have been made and the team approves your article, then your article will be published.
```

Comment:
Can you add some steps about the CI pipeline as well? We use Vercel for deployment and will add some GH actions. So just mention that the checks should pass and they can make their PR as a `draft` unless all the checks pass. If its failing and they cannot debug it, let us know

Reply:
Yes I will do so and push my changes by tomorrow